gitlab/config/201-clusterrole.yaml

```diff
@@ -0,0 +1,105 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: gitlabsource-manager-role
+rules:
+- apiGroups:
+  - source.knative.dev
+  resources:
+  - gitlabsources
+  verbs: &everything
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - source.knative.dev
+  resources:
+  - gitlabsources/status
+  verbs:
+  - get
```

Comment:
Under what circumstance would you `get` just the `/status` sub-resource? (apologies if I overlooked something)

Reply:
up